apps/studio/package.json

```diff
@@ -131,7 +131,7 @@
     "kysely": "^0.27.3",
     "lodash": "^4.17.21",
     "nanoid": "^5.0.9",
-    "next": "^15.5.7",
+    "next": "^15.5.8",
```

Comment:
Shouldn't this be 15.5.9 as per https://nextjs.org/blog/security-update-2025-12-11?